.gitignore

```diff
@@ -3,7 +3,7 @@
 imgui.ini
 cmake-*
 ImportExecutables-*.cmake
-out
+out*
```

Comment:
Well, this was for my convenience, as when I needed multiple build directories I created out, out2, out3, so that is why I changed it to out*, but I mistakenly duplicated it with /out* at the bottom of the file.  So I'm removing the bottom addition and changing to /out*.